src/drivers/esp32/src/esp_rom_bootloader.c

```diff
@@ -54,3 +54,29 @@ void espRomBootloaderInit()
   pinMode(DECK_GPIO_IO1, INPUT_PULLUP);
 }
 
+bool espRomBootloaderSync(uint8_t *sendBuffer)
+{
+  senderPacket.command = SYNC;
+  senderPacket.dataSize = 0x24;
+  sendBuffer[9 + 0] = 0x07;
+  sendBuffer[9 + 1] = 0x07;
+  sendBuffer[9 + 2] = 0x12;
+  sendBuffer[9 + 3] = 0x20;
+  for (int i = 0; i < 32; i++)
+  {
+    sendBuffer[9 + 4 + i] = 0x55;
```

Comment:
Are these values obvious to you? Do one need to understand what they mean? Where can you find an explanation? Would it benefit with a comment to explain how this sync is done?

Reply:
I added a definition for this index